recipes/cdflib/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "cdflib" %}
+{% set version = "0.3.17" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9b027b0e5b7d84816aafa13edc5df3767c560881e8a93348f92cdfba29d7897a
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
```

Comment:
Looks like "cdflb" is Python 3, so please use `python >=3.5` here.

Reply:
Looks like you need to add `numpy` in `run` as well. In general, everything needed to run the package needs to go to the `run` section.